great_expectations/cli/toolkit.py

```diff
@@ -84,9 +89,12 @@ def get_or_create_expectation_suite(
             batch_request=batch_request,
         )
         while True:
-            expectation_suite_name = click.prompt(
-                "\nName the new Expectation Suite",
-                default=default_expectation_suite_name,
+            expectation_suite_name = cast(
```

Comment:
@Kilo59 For what it is worth, in my environment, `PyCharm` is not showing any warnings for the way it currently is.

Reply:
`mypy` should raise a warning with any redundant casts.
https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L24